packages/integrations/svelte/src/index.ts

```diff
@@ -37,9 +40,14 @@ async function getViteConfiguration({
 }: ViteConfigurationArgs): Promise<UserConfig> {
 	const defaultOptions: Partial<Options> = {
 		emitCss: true,
-		compilerOptions: { dev: isDev, hydratable: true },
+		compilerOptions: { dev: isDev },
 	};
 
+	if (!isSvelte5) {
+		// @ts-ignore
+		defaultOptions.compilerOptions.hydratable = true;
```

Comment:
Interesting! Hydration is removed in Svelte 5? Is there any way to use hydratable Svelte components (4.0 or earlier) in a Svelte 5 project?

Reply:
Hydratable is enabled by default in Svelte 5. If you pass `hydratable: true`, it'll show a warning that you don't have to pass it now. I'll add a note here.